### ISSUE TITLE
Not working in Firefox when logged in to YouTube

### ISSUE THREAD

**Issue by R3G3N3R4T0R** [yakisova41/return-youtube-comment-username#43]
The script does not work when logged in and using Firefox. It works fine in Chromium-based browsers.
The culprit is querySelector from the function [rewriteCommentNameFromContinuationItems](https://github.com/yakisova41/return-youtube-comment-username/blob/3c216312d4f5874bf23ae8828cb2c8a37bdfd22f/src/index.ts#L286)
```ts
const commentRenderer = commentElem?.querySelector(
  "ytd-comment-renderer"
);
```

Debugger Logged Out
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/54089608/834ec91d-dcf2-4552-bc5e-58bd98be1a5b)
Debugger Logged In
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/54089608/a0e51f1b-86e1-4aed-8f4c-cf8776909b3c)
HTML Logged In
![image](https://github.com/yakisova41/return-youtube-comment-username/assets/54089608/9c3053cd-372e-4950-8440-2ae8451d2309)

These are from the same link, one successfully selected `ytd-comment-renderer` from `ytd-comment-thread-renderer` and another failed to do so. And `ytd-comment-renderer` does exist in the logged in version.

CY Fung's alternative implementation also has the same issue. I haven't debugged his but also no request for username when logged in so likely the same problem.

### Versions Tested
- v0.3.2.1 to v0.3.4 (Firefox 114.0.2/TamperMonkey v4.18.1)
- V0.3.4 (Firefox 114.0.2/ViolentMonkey v2.14.0)
- v0.3.4 (Librewolf 110.0.1/ViolentMonkey v2.14.0)

**Comment by yakisova41** [comment]
Am I correct that `ytd-comment-renderer` is null when I am logged into my Google account?🤨
It is working fine in my Firefox, so I would like to know the detailed procedure to reproduce it.

**Comment by R3G3N3R4T0R** [comment]
No, `ytd-comment-renderer` is not null when logged in if the tag is what you are referring to. it is just that the `commentElem.querySelector("ytd-comment-renderer")` method returned null when logged in.
It appears that both my main and alt are affected so if anything differs between our systems it could be a regional issue. YouTube did not use handles when logged in, only when logged out until a few days ago for me.

Step to reproduce it is as such:
1. Create a new profile in Firefox, load in TamperMonkey/ViolentMonkey and your script from GreasyFork.
2. open www.youtube.com and pick a video.
3. scroll down and YouTube comments should have usernames
4. Log in to YouTube, and return to the same video
5. scroll down and YouTube comments should have handles.

**Comment by yakisova41** [comment]
We have identified the cause and have released a fixed version v0.3.5.
Please update it.

**Comment by R3G3N3R4T0R** [comment]
Can confirm it is fixed. Thanks for the quick response
